packages/opentelemetry-api/src/context/context.ts

```diff
@@ -38,49 +36,45 @@ const SUPPRESS_INSTRUMENTATION_KEY = createContextKey(
 const BAGGAGE_KEY = createContextKey('OpenTelemetry Baggage Key');
 
 /**
- * Return the active span if one exists
+ * Return the span if one exists
  *
  * @param context context to get span from
  */
-export function getActiveSpan(context: Context): Span | undefined {
-  return (context.getValue(ACTIVE_SPAN_KEY) as Span) || undefined;
+export function getSpan(context: Context): Span | undefined {
+  return (context.getValue(SPAN_KEY) as Span) || undefined;
 }
 
 /**
- * Set the active span on a context
+ * Set the span on a context
  *
  * @param context context to use as parent
  * @param span span to set active
  */
-export function setActiveSpan(context: Context, span: Span): Context {
-  return context.setValue(ACTIVE_SPAN_KEY, span);
+export function setSpan(context: Context, span: Span): Context {
```

Comment:
what if we name it `setSpanInContext` and for get we will have `getSpanFromContext`

Reply:
If we go this path we should do the same with e.g. `setBaggage`.
But I think it would be more user friendly to use a namespace to get something like `context.setSpan()`, `context.getBaggage()`,... (Refs: #1750).
Unfortunately API exports already `context` and `Context` so adding one more would be really confusing.

Reply:
Well the problem with `context.getBagage()` is that the `Context` goal was to be de-coupled from the usage of it (what we store inside).

> what if we name it setSpanInContext and for get we will have getSpanFromContext

I don't think we should add `inContext`/`fromContext` since its already how these methods works (ie: setting value in the context).

Reply:
>I don't think we should add inContext/fromContext since its already how these methods works (ie: setting value in the context).

I agree. `api.context.setSpan`/`api.context.setBaggage`/`api.context.getBaggage`/etc are sufficiently clear IMO.

>But I think it would be more user friendly to use a namespace to get something like context.setSpan(), context.getBaggage(),... (Refs: #1750).

The way you implemented this PR is already `api.context.setSpan`. I'm confused what you mean by adding additional namespacing.

Reply:
> The way you implemented this PR is already api.context.setSpan. I'm confused what you mean by adding additional namespacing.

Not exactly, `setSpan` etc are only exported as utils so you must do:
```ts
import { setSpan, context } from '@opentelemetry/api'
setSpan(api.context.active(), span)
```

instead of what he suggested (if i correctly understood):
```ts
import { context } from '@opentelemetry/api'
context.active().setSpan(span)
```

Reply:
for me `setSpan` as an util it confusing as the first things that you can ask is "set span where on what". That's why fi we have a "standalone" function I would be very precise here and then have `setSpanInContext` or `getSpanFromContext`.  At the same time if we always require a context when trying to set a span, I don't like this
```
   setSpan(someContext, someSpan); 
```
and would rather have 
```
  someContext.setSpan(someSpan);
```
But having a standalone util function `setSpan` is not very intuitive for the user.

Reply:
Can we add those methods on the context since they aren't in the specs ? I looked around in go, java or python and they don't seems to have this

Reply:
> Can we add those methods on the context since they aren't in the specs ? I looked around in go, java or python and they don't seems to have this

probably not anymore, especially shortly before GA

Reply:
I created a dedicated issue regarding namespacing: #1750
The main point of this PR is to get rid of the `active`, `Extracted` and `Parent` parts in these APIs as they are wrong in my opinion.
But sure, I'm fine with moving into a different direction or let another PR eat this one.

Reply:
> Can we add those methods on the context since they aren't in the specs ? I looked around in go, java or python and they don't seems to have this

The alternative of these utils is most likely to export the corresponding context keys and tell users to use the context APIs with them. But this doesn't solve the question if these keys should be exported flat on API or on context.

Reply:
> The alternative of these utils is most likely to export the corresponding context keys and tell users to use the context APIs with them

I don't think thats a viable alternative since it will allows for users to insert anything into the context for a span, where as now we can typecheck that they are using a `Span`.